config-rs/tree/master/examples/simple doesn't compile because std::collections doesn't have Map

# Problem

When trying to build application it fails with error:
```
error[E0432]: unresolved import `std::collections::Map`
 --> src/main.rs:2:5
  |
2 | use std::collections::Map;
  |     ^^^^^^^^^^^^^^^^^^^^^ no `Map` in `collections`
```
# Steps to reproduce

1. Copy code from `config-rs/tree/master/examples/simple`
2. Run `cargo build`
3. Get error

# Suggested solution

Replace `std::collections::Map` with `std::collections::HashMap`

--- a/.github/workflows/msrv.yml
+++ b/.github/workflows/msrv.yml
@@ -137,3 +137,30 @@ jobs:
         with:
           command: clippy
           args: -- -D warnings
+
+  check-examples:
+    name: Check examples
+    needs: [check]
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rust:
+          - 1.46.0
+          - stable
+
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      - name: Install toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: ${{ matrix.rust }}
+          override: true
+
+      - name: Run cargo check
+        uses: actions-rs/cargo@v1
+        with:
+          command: check
+          args: --examples
+

--- a/examples/glob/src/main.rs
+++ b/examples/glob/src/main.rs
@@ -1,5 +1,5 @@
 use std::path::Path;
-use std::collections::Map;
+use std::collections::HashMap;
 use config::*;
 use glob::glob;
 
@@ -14,9 +14,9 @@ fn main() {
         .merge(File::from(Path::new("conf/05-some.yml"))).unwrap()
         .merge(File::from(Path::new("conf/99-extra.json"))).unwrap();
 
-    // Print out our settings (as a Map)
+    // Print out our settings (as a HashMap)
     println!("\n{:?} \n\n-----------",
-             settings.try_into::<Map<String, String>>().unwrap());
+             settings.try_into::<HashMap<String, String>>().unwrap());
 
     // Option 2
     // --------
@@ -28,9 +28,9 @@ fn main() {
                     File::from(Path::new("conf/99-extra.json"))])
         .unwrap();
 
-    // Print out our settings (as a Map)
+    // Print out our settings (as a HashMap)
     println!("\n{:?} \n\n-----------",
-             settings.try_into::<Map<String, String>>().unwrap());
+             settings.try_into::<HashMap<String, String>>().unwrap());
 
     // Option 3
     // --------
@@ -43,7 +43,7 @@ fn main() {
                    .collect::<Vec<_>>())
         .unwrap();
 
-    // Print out our settings (as a Map)
+    // Print out our settings (as a HashMap)
     println!("\n{:?} \n\n-----------",
-             settings.try_into::<Map<String, String>>().unwrap());
+             settings.try_into::<HashMap<String, String>>().unwrap());
 }

--- a/examples/simple/src/main.rs
+++ b/examples/simple/src/main.rs
@@ -1,4 +1,4 @@
-use std::collections::Map;
+use std::collections::HashMap;
 
 fn main() {
     let mut settings = config::Config::default();
@@ -9,7 +9,7 @@ fn main() {
         // Eg.. `APP_DEBUG=1 ./target/app` would set the `debug` key
         .merge(config::Environment::with_prefix("APP")).unwrap();
 
-    // Print out our settings (as a Map)
+    // Print out our settings (as a HashMap)
     println!("{:?}",
-             settings.try_into::<Map<String, String>>().unwrap());
+             settings.try_into::<HashMap<String, String>>().unwrap());
 }

--- a/examples/watch/src/main.rs
+++ b/examples/watch/src/main.rs
@@ -1,5 +1,5 @@
 use config::*;
-use std::collections::Map;
+use std::collections::HashMap;
 use std::sync::RwLock;
 use notify::{RecommendedWatcher, DebouncedEvent, Watcher, RecursiveMode};
 use std::sync::mpsc::channel;
@@ -20,7 +20,7 @@ fn show() {
                  .read()
                  .unwrap()
                  .clone()
-                 .try_into::<Map<String, String>>()
+                 .try_into::<HashMap<String, String>>()
                  .unwrap());
 }
 

